pkg/tracing/basictracing/recorder_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package tracing
+package basictracing
```

Comment:
so, this is package `pkg/tracing/basictracing` ?  is there a better name?  would even `pkg/tracing/basic` work?

and what is meant by `basic`?